ui/app/components/pages/first-time-flow/select-action/select-action.component.js

```diff
@@ -0,0 +1,104 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import Button from '../../../button'
+import { INITIALIZE_CREATE_PASSWORD_ROUTE, INITIALIZE_NOTICE_ROUTE, INITIALIZE_IMPORT_WITH_SEED_PHRASE_ROUTE } from '../../../../routes'
```

Comment:
Nitpick: can we split this import onto multiple lines?

Reply:
Fixed in ac872717b